llvm/test/tools/llvm-objcopy/ELF/set-symbol-visibility-check-errors.test

```diff
@@ -0,0 +1,36 @@
+# RUN: yaml2obj %s -o %t.o
```

Comment:
You've still got thiss and the undef case in separate files unnecessarily. Please fold them into the ain test file.